packages/open-payments/src/client/index.ts

```diff
@@ -93,6 +93,7 @@ export interface CreateAuthenticatedClientArgs
   extends CreateUnauthenticatedClientArgs {
```

Comment:
Kind of outside the scope of this PR, but I'm a bit confused why TS doesn't complain about
```ts
const axiosInstance = createAxiosInstance({
    privateKey: args.privateKey,
    keyId: args.keyId,
    requestTimeoutMs:
      args?.requestTimeoutMs ?? config.DEFAULT_REQUEST_TIMEOUT_MS
}
```
when privateKey and keyId are optional in `createUnauthenticatedClient`.

Reply:
I think this due to
https://github.com/interledger/rafiki/blob/2f17be36bb705e3afbe449d577d380e79e876646/tsconfig.build.json#L28
which was toggled in
- https://github.com/interledger/rafiki/pull/758

@matdehaast @traviscrist Do either of you remember why we disabled strict type checking?

Reply:
>If you enable it how many issues does it find?

tbd
- https://github.com/interledger/rafiki/pull/768